<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="windowTitle">
   <string>MA5 扫描器（东财接口）</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayoutMain">
    <item>
     <widget class="QListWidget" name="listMenu">
      <property name="minimumSize">
       <size>
        <width>160</width>
        <height>0</height>
       </size>
      </property>
      <item>
       <property name="text">
        <string>MA5筛选</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>MACD筛选</string>
       </property>
      </item>
     </widget>
    </item>
    <item>
     <widget class="QStackedWidget" name="stackedWidget">
      <widget class="QWidget" name="pageMa5">
       <layout class="QVBoxLayout" name="verticalLayoutMa5">
        <item>
         <layout class="QGridLayout" name="gridLayout">
          <item row="0" column="0">
           <widget class="QPushButton" name="btnScan">
            <property name="text">
             <string>开始扫描</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QPushButton" name="btnCancel">
            <property name="text">
             <string>取消</string>
            </property>
           </widget>
          </item>
          <item row="0" column="2">
           <widget class="QPushButton" name="btnExport">
            <property name="text">
             <string>导出CSV</string>
            </property>
           </widget>
          </item>

          <item row="1" column="0">
           <widget class="QLabel" name="label_1">
            <property name="text">
             <string>N天(收盘&lt;MA5):</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QSpinBox" name="spinBelowDays">
            <property name="minimum"><number>1</number></property>
            <property name="maximum"><number>20</number></property>
            <property name="value"><number>3</number></property>
           </widget>
          </item>
          <item row="1" column="2">
           <widget class="QCheckBox" name="cbIncludeBJ">
            <property name="text"><string>包含北交所</string></property>
            <property name="checked"><bool>true</bool></property>
           </widget>
          </item>
          <item row="1" column="3">
           <widget class="QCheckBox" name="cbExcludeST">
            <property name="text"><string>过滤ST</string></property>
            <property name="checked"><bool>true</bool></property>
           </widget>
          </item>

          <item row="2" column="0">
           <widget class="QLabel" name="label_2">
            <property name="text"><string>并发:</string></property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QSpinBox" name="spinInFlight">
            <property name="minimum"><number>1</number></property>
            <property name="maximum"><number>50</number></property>
            <property name="value"><number>12</number></property>
           </widget>
          </item>
          <item row="2" column="2">
           <widget class="QLabel" name="label_3">
            <property name="text"><string>分页大小:</string></property>
           </widget>
          </item>
          <item row="2" column="3">
           <widget class="QSpinBox" name="spinPageSize">
            <property name="minimum"><number>100</number></property>
            <property name="maximum"><number>1000</number></property>
            <property name="singleStep"><number>100</number></property>
            <property name="value"><number>500</number></property>
           </widget>
          </item>

          <item row="3" column="0">
           <widget class="QLabel" name="label_4">
            <property name="text"><string>超时(ms):</string></property>
           </widget>
          </item>
          <item row="3" column="1">
           <widget class="QSpinBox" name="spinTimeout">
            <property name="minimum"><number>3000</number></property>
            <property name="maximum"><number>60000</number></property>
            <property name="singleStep"><number>1000</number></property>
            <property name="value"><number>12000</number></property>
           </widget>
          </item>
          <item row="3" column="2">
           <widget class="QLabel" name="label_5">
            <property name="text"><string>重试:</string></property>
           </widget>
          </item>
          <item row="3" column="3">
           <widget class="QSpinBox" name="spinRetry">
            <property name="minimum"><number>0</number></property>
            <property name="maximum"><number>5</number></property>
            <property name="value"><number>2</number></property>
           </widget>
          </item>

          <item row="4" column="0">
           <widget class="QLabel" name="label_6">
            <property name="text"><string>排序字段:</string></property>
           </widget>
          </item>
          <item row="4" column="1">
           <widget class="QComboBox" name="comboSortField">
            <item>
             <property name="text">
              <string>偏离(%)</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>偏离绝对值</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>市盈率(PE)</string>
             </property>
            </item>
           </widget>
          </item>
          <item row="4" column="2">
           <widget class="QCheckBox" name="cbSortDesc">
            <property name="text"><string>降序</string></property>
            <property name="checked"><bool>true</bool></property>
           </widget>
          </item>

          <item row="0" column="3">
           <layout class="QHBoxLayout" name="horizontalLayout">
            <item>
             <widget class="QLineEdit" name="editTestCode">
              <property name="placeholderText"><string>测试代码/secid：如 600000 或 1.600000</string></property>
              <property name="text"><string>600000</string></property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="btnTest">
              <property name="text"><string>测试接口</string></property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>

        <item>
         <widget class="QProgressBar" name="progressBar"/>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QLabel" name="labelStage">
            <property name="text"><string>就绪</string></property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer">
            <property name="orientation"><enum>Qt::Horizontal</enum></property>
            <property name="sizeHint" stdset="0">
             <size><width>40</width><height>20</height></size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QLabel" name="labelProgress">
            <property name="text"><string>0/0</string></property>
           </widget>
          </item>
         </layout>
        </item>

        <item>
         <widget class="QTableView" name="tableView"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="pageMacd">
       <layout class="QVBoxLayout" name="verticalLayoutMacd">
        <item>
         <widget class="QLabel" name="labelMacdIntro">
          <property name="text">
           <string>MACD 筛选（开发中）：后续将加入 DIF/DEA/柱体条件筛选。</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="verticalSpacerMacd">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
